dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/web/ContentletWebAPIImpl.java

```diff
@@ -739,10 +742,9 @@ private void handleEventRecurrence(Map<String, Object> contentletFormData, Conte
 				contentlet.setBoolProperty("recurs",false);
 			}
 
-
 	}
 
-	private void _populateContent(Map<String, Object> contentletFormData,
+	private Contentlet _populateContent(Map<String, Object> contentletFormData,

```

Comment:
rename the method to be align with the java convension

Reply:
changed